robotiq_c_model_control/src/robotiq_c_model_control/robotiq_gripper_action_interface.py

```diff
@@ -140,6 +140,11 @@ def shutdown_as(self):
             self.gripper_as_.cleanup()
         self.gripper_as_ = None
 
+    def wait_for_next_status(self, seconds=0.25):
+        with self.status_cv:
+            self.status_cv.wait(seconds)
+            return deepcopy(self.last_status)
```

Comment:
deepcopy is slow and should be avoided if you know the type and don't need genericity

Reply:
@mechterp10 built this code originally with deepcopy, and I had assumed it was necessary to ensure that the retrieved status remains consistent. Since ```last_status``` is updated in a separate thread, a simple reference to it isn't good enough, that is unless it were protected by some other means. I'll chat with Josh at some point today.